src/app/app.component.ts

```diff
@@ -22,4 +22,8 @@ export class AppComponent {
   onSave(fhirResult): void {
     this.fhirPreview = JSON.stringify(fhirResult, null, 2);
   }
+
+  addTotalScoreRule(): void {
+    const result = this.ruleEditorService.addTotalScoreRule(fhir.phq9, context);
```

Comment:
"context" is a very generic term which has meanings in FHIR.  If this is a linkId, please rename the variable to that.  (I know that variable was not created in this PR, but please change it.)

Reply:
Fixed